### PR TITLE
Make the user log in before seeing the bank invoice.

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
- ditto.

#### What problem is this solving?
- The checkout doesn't return the correct url when the user isn't indentified. Therefore we need to make him log in first before he can see the bank invoice it self.

#### How should this be manually tested?

https://felipe--boticario.myvtex.com/checkout/orderPlaced/?og=v83637215bot

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/utils/index.tsx [hunk 1]
```diff
@@ -188,3 +188,14 @@ export function orderSplitMessage({
     takeaways,
   })
 }
+
+export function parseBankInvoiceUrl(url: string) {
+  const isEncrypted = url.match(/(\*.\*.)+\*\w\*/g)
+
+  if (!isEncrypted) return url
+  if (!window || !window.location) return ''
```
Comment (kaisermann) [inline react/utils/index.tsx]: Why is this needed?

Comment (klzns) [reply react/utils/index.tsx]: SSR I think

File: react/utils/index.tsx [hunk 2]
```diff
@@ -188,3 +188,14 @@ export function orderSplitMessage({
...
+
+  return `/login?returnUrl=${encodeURIComponent(
```
Comment (klzns) [inline react/utils/index.tsx]: I think there's rootPath:
https://vtex.slack.com/archives/CD0C3M3MJ/p1565129756184500

For dealing with stores like Samsung AR

So you might need to append some string here

File: react/utils/index.tsx [hunk 3]
```diff
@@ -188,3 +188,14 @@ export function orderSplitMessage({
...
   })
 }
+
+export function parseBankInvoiceUrl(url: string) {
+  const isEncrypted = !!url.match(/(\*.\*.)+\*\w\*/g)
+
+  if (!isEncrypted) return url
+  if (!window || !window.location) return ''
+
+  return `/login?returnUrl=${encodeURIComponent(
```
Comment (klzns) [inline react/utils/index.tsx]:
```suggestion
  return `${__RUNTIME__.rootPath || ''}/login?returnUrl=${encodeURIComponent(
```

Comment (arthurepc) [reply react/utils/index.tsx]: In order to access the runtime like this, I think you have to use the window. Like: `window.__RUNTIME__` or you could use the render HOC `withRuntimeContext`. I think the latter is more intuitive
